app/routes.js

```diff
@@ -18,7 +18,7 @@ router.get('/', async (req, res) => {
   res.locals.payNumberOfOrganisations = payData['numberOfOrganisations']
 
   const notifyData = await fetchData('https://www.notifications.service.gov.uk/features/performance.json', 'notifyDataCache')
-  console.log(notifyData)
```

Comment:
Did you want to delete this?

Reply:
@cadmiumcat yep! It was debug I forgot to take out